app/src/main/kotlin/com/wire/android/ui/authentication/welcome/WelcomeScreen.kt

```diff
@@ -161,7 +170,10 @@ private fun WelcomeContent(
                     horizontal = MaterialTheme.wireDimensions.welcomeButtonHorizontalPadding
                 )
             ) {
-                LoginButton() { navigate(NavigationCommand(LoginScreenDestination())) }
+                LoginContent(
+                    maxAccountsReached = maxAccountsReached,
+                    onLoginButtonActionClicked = { navigate(NavigationCommand(LoginScreenDestination())) }
+                )
```

Comment:
As passing `maxAccountsReached` to the login content, could it also be done for creating an account? This way if the users tries to login it gets the popup dialog, but how does the flow goes when creating a new account as we login after account creation? 🤔 just thinking of other possible ways an user can try to have more than max accounts

Reply:
good catch, solved !